virttest/qemu_vm.py

```diff
@@ -1599,6 +1600,38 @@ def add_secure_guest_descriptor(params):
             for k, v in backend_props.get(sectype, {}):
                 machine_dev.set_param(k, v)
 
+        def __conflict_check(params):
+            """
+            Based on the params, check if it's a conflict.
+
+            :param params: A dict containing VM params
+            :type params: dict
+            """
+            iothread_scheme = params.get("iothread_scheme")
+            image_iothread_vq_mapping = params.get(
+                "image_iothread_vq_mapping")
+            iothreads_lst = params.objects("iothreads")
+            """
+            The legacy 'iothread_scheme' does NOT support the
+            parameter 'iothread_vq_mapping'. If you're going to use the legacy
+            'iothread_scheme', the 'iothread_vq_mapping' must NOT be set.
+            """
+            if (iothread_scheme and image_iothread_vq_mapping) or (
```

Comment:
No need to check iothread_scheme and image_iothread_vq_mapping conflict here, as the former is a global param while the latter is a image level param, this check is covered by line 1628

Reply:
> No need to check iothread_scheme and image_iothread_vq_mapping conflict here, as the former is a global param while the latter is a image level param, this check is covered by line 1628

Based on my understanding, `image_iothread_vq_mapping` and `image_iothread_vq_mapping_xxx` are two parameters in Cartesian list.

Reply:
> > No need to check iothread_scheme and image_iothread_vq_mapping conflict here, as the former is a global param while the latter is a image level param, this check is covered by line 1628
> 
> Based on my understanding, `image_iothread_vq_mapping` and `image_iothread_vq_mapping_xxx` are two parameters in Cartesian list.

literal difference. an image's image_iothread_vq_mapping will be set to params['image_iothread_vq_mapping'] if params['image_iothread_vq_mapping_xxx'] is not set. So 1628 can cover the check. 

If images is not set, and image_iothread_vq_mapping is set, this check makes sense, for users may want to hotplug images. 

Then let's keep it